src/components/Feed/Feed.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * React Starter Kit (https://www.reactstarterkit.com/)
+ *
+ * Copyright © 2014-2016 Kriasoft, LLC. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+import React, {PropTypes} from 'react'
+import withStyles from 'isomorphic-style-loader/lib/withStyles'
+import s from './Feed.scss'
+import Story from '../Story'
+import StoryInput from '../StoryInput'
+import {connect} from 'react-redux'
+
+function Feed ({stories}) {
+  return (
+    <div className={ s.root }>
+      <div className={ s.container }>
+        <StoryInput />
+        <ul className={ s.stories }>
+          { stories.map((item, index) => (
```

Comment:
Create a function that returns all the Stories. The function can be placed in this same file. Improves readability and separate concerns.